batect.yml

```diff
@@ -1,4 +1,3 @@
-forbid_telemetry: true
```

Comment:
Forbid telemetry can only be on the root config file.

Reply:
That's OK, we can re-enable telemetry for consistency. I had this when it was a closed-source tool because I'm paranoid.